hi-ml-azure/src/health/azure/azure_util.py

```diff
@@ -869,3 +872,24 @@ def upload_to_datastore(datastore_name: str, local_data_folder: Path, remote_pat
     datastore.upload(str(local_data_folder), target_path=str(remote_path), overwrite=overwrite,
                      show_progress=show_progress)
     logging.info(f"Uploaded data to {str(remote_path)}")
+
+
+def download_checkpoints_from_run(run_id: str, checkpoint_dir: str, output_path: Path,
```

Comment:
output_path -> output_folder?

Reply:
the naming is now somewhat inconsistent. it is "download_checkpoint_from_run", but "download_run_files_from_run_id".  While not many people are using that, maybe clean up the naming? I'd go with `download_files_from_run`. I expect that most people will use the flavour of download where a run_id given, and give the other functions that take Run objects as arguments more complicated names.